Library/Homebrew/cli_parser.rb

```diff
@@ -34,7 +37,7 @@ def comma_array(name, description: nil)
         end
       end
 
-      def flag(name, description: nil)
+      def flag(name, description: nil, required_for: nil, depends_on: nil)
```

Comment:
Can just omit the `nil`s I think and they will be defaulted.

Reply:
Seems like `nil` can't be omitted, then they become mandatory named arguments.

Reply:
Ah, gotcha, ignore me then!